src/test/java/seedu/algobase/logic/commands/HelpCommandTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.algobase.logic.commands;
+
+import static seedu.algobase.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.algobase.testutil.Assert.assertThrows;
+import static seedu.algobase.testutil.TypicalProblems.getTypicalAlgoBase;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.algobase.model.Model;
+import seedu.algobase.model.ModelManager;
+import seedu.algobase.model.UserPrefs;
+
+class HelpCommandTest {
+
+    private static final String EXPECTED_COMMAND_LIST = "Available commands are: [add, addplan, addtag, "
+        + "addtag, clear, delete, deleteplan, deletetag, deletetask, donetask, edit, editplan, edittag, exit, "
+        + "find, findplan, help, list, listplan, listtag, sort, switch, undonetask]\n"
```

Comment:
Hmm hardcoded string? I guess it is not a good practice?

Reply:
Do you have any suggestions on writing this? Is hardcoded string in tests a bad idea?